react/features/settings/components/web/SettingsDialog.js

```diff
@@ -56,6 +62,116 @@ type Props = {
     dispatch: Function
 };
 
+/**
+ * Creates the styles for the component.
+ *
+ * @param {Object} theme - The current UI theme.
+ *
+ * @returns {Object}
+ */
+const styles = theme => {
+    return {
+        root: {
+            display: 'flex',
+            width: '100%',
+
+            '&.profile-pane': {
+                flexDirection: 'column'
+            },
+
+            '& .auth-name': {
+                marginBottom: theme.spacing(1)
+            },
+
+            '& .calendar-tab, & .device-selection': {
+                marginTop: '20px'
+            },
+
+            '& .mock-atlaskit-label': {
+                color: '#b8c7e0',
+                fontSize: '12px',
+                fontWeight: 600,
+                lineHeight: 1.33,
+                padding: `20px 0px ${theme.spacing(1)}px 0px`
+            },
+
+            '& input[type="checkbox"]:checked + svg': {
+                '--checkbox-background-color': '#6492e7',
+                '--checkbox-border-color': '#6492e7'
+            },
+
+            '& input[type="checkbox"] + svg + span': {
+                color: '#9FB0CC'
+            },
+
+            [[ '& .calendar-tab',
+                '& .more-tab',
+                '& .box' ]]: {
+                display: 'flex',
+                justifyContent: 'space-between',
+                width: '100%'
+            },
+
+            '& .profile-edit': {
+                display: 'flex',
+                width: '100%'
+            },
+
+            '& .profile-edit-field': {
+                flex: 0.5,
+                marginRight: '20px'
```

Comment:
This wasn't in the SCSS. Is this intentional?

Reply:
There are two `profile-edit-field` classes. Setting `flex` value [here](https://github.com/jitsi/jitsi-meet/blob/4db7312d53223c1e8aa5e9d365e835728d9f40f6/css/modals/settings/_settings.scss#L50) and setting margin [here](https://github.com/jitsi/jitsi-meet/blob/4db7312d53223c1e8aa5e9d365e835728d9f40f6/css/modals/settings/_settings.scss#L73)